cassandra/README.md

```diff
@@ -7,12 +7,125 @@ Get metrics from cassandra service in real time to:
 * Visualize and monitor cassandra states
 * Be notified about cassandra failovers and events.
 
+Learn more about how to monitor Cassandra performance metrics thanks to [our series of posts](https://www.datadoghq.com/blog/how-to-monitor-cassandra-performance-metrics/). We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Cassandra.
+  		  
+For information on JMX Checks, please see [here](http://docs.datadoghq.com/integrations/java/).
+  
 # Installation
 
 The Cassandra check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your Cassandra nodes.
 
 # Configuration
 
+To capture Cassandra metrics you need to install the Datadog Agent. Metrics will be captured using a JMX connection. 
+We recommend the use of Oracle's JDK for this integration. 
+
+This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the info page. You can specify the metrics you are interested in by editing the configuration below. To learn how to customize the metrics to collect visit the [JMX Checks documentation](https://docs.datadoghq.com/integrations/java/) for more detailed instructions. If you need to monitor more metrics, please send us an email at support@datadoghq.com
+
+1. Configure the Agent to connect to Cassandra, just edit `conf.d/cassandra.yaml`:
+
+```
```

Comment:
I'd agree that we likely dont need the whole yaml file here, or if we do it should be inserted via a macro rather than being duplicated here.

@kmshultz what do you think?